core-bundle/contao/templates/twig/content_element/element_group.html.twig

```diff
@@ -0,0 +1,7 @@
+{% extends "@Contao/content_element/_base.html.twig" %}
```

Comment:
We should probably move or rename this file, because there is no *element_group* content element. 

Here are some options:
1. `content_element/_element_group.html.twig` to make it a partial like `_video.html.twig`.
2. maybe even `component/_element_group.html.twig`
3. `content_element/accordion/_element_group.html.twig` if the partial should only belong to the accordion. For this we would also need to adjust the template option listener (or the Twig finder) to exclude templates starting with an underscore, which would probably be a nice behavior anyways.

Reply:
> because there is no _element_group_ content element.

What do you mean by that?

<img width="537" alt="" src="https://github.com/contao/contao/assets/1192057/27684397-a5fe-41ec-b17f-a4e0e093bc45">

<img width="607" alt="" src="https://github.com/contao/contao/assets/1192057/5acce4bb-9b61-4168-943e-7ce92913ea13">

Reply:
Ah, got it. Sorry, I misunderstood the concept.

So an "element group" is a generic group that supports any children. In the future, when the nested elements concept will be used more heavily, there will likely be the need to allow only certain children and maybe define a maximum amount (say accordions can only have `text` and up to 5 elements). I think these constraints will be bound to the wrappers as they are quite specific for the use case. How should we deal with this if the element group is its own element? :thinking:

Reply:
> In the future, when the nested elements concept will be used more heavily, there will likely be the need to allow only certain children and maybe define a maximum amount (say accordions can only have `text` and up to 5 elements).

I don‘t think so. If someone wants to nest an accordion inside another accordion, they should be able to do so.

> I think these constraints will be bound to the wrappers as they are quite specific for the use case. How should we deal with this if the element group is its own element?

What is the point of having an "accordion element group" element, a "nested tabs element group" element and a "slider element group" element that all do the exact same thing? A generic element makes more sense to me.

Reply:
I think a general “element group” element should be fine and can also be used for other use-cases.

> How should we deal with this if the element group is its own element? 🤔

I think in the long run we will probably need to be able to also allow/disallow elements more than one level deep, so this could still be achieved using a general group element.

Reply:
> I don‘t think so. If someone wants to nest an accordion inside another accordion, they should be able to do so.

In most of the "design heavy" projects I had, you would only style certain things (based on the design of an agency for instance) and also have editors that are using everything available to them. So IMHO that's pretty common.